CONTRIBUTING.md

```diff
@@ -21,6 +21,31 @@ Contributions will be distributed with Vim under the Vim license.  Providing a
 change to be included implies that you agree with this and your contribution
 does not cause us trouble with trademarks or patents.  There is no CLA to sign.
 
+## Signing-off commits
+
+While not required, it's recommended to use **Signed-off commits** to ensure
+transparency, accountability, and compliance with open-source best practices.
+Signed-off commits follow the [Developer Certificate of Origin (DCO)][15],
+which confirms that contributors have the right to submit their changes under
+the project's license.  This process adds a `Signed-off-by` line to commit
+messages, verifying that the contributor agrees to the project's licensing
+terms.  To sign off a commit, simply use the -s flag when committing:
+
+```sh
+git commit -s -m "Your commit message"
```

Comment:
I don't like recommending `-m`, so how about

```suggestion
git commit -s
```

?

Otherwise looks great to me.

Reply:
fixed